sdk/communication/azure-communication-rooms/swagger/README.md

```diff
@@ -16,9 +16,10 @@ autorest SWAGGER.md
 ### Settings
 
 ```yaml
-tag: package-rooms-2021-04-07
```

Comment:
No need to tag the generated files for version anymore?

Reply:
Yeah because the tag is defined in this file we refer https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e30976f6ccb058a36cd2f9d5160e1fd51f6c5d95/specification/communication/data-plane/Rooms/readme.md